packages/common/src/account/kms/kmsKeyToAccount.ts

```diff
@@ -19,11 +20,15 @@ export type KmsAccount = LocalAccount<"aws-kms"> & {
  * @returns A Local Account.
  */
 export async function kmsKeyToAccount({
-  keyId,
+  keyId = process.env.AWS_KMS_KEY_ID,
```

Comment:
I don't think this will work with how bundlers work - they generally use env vars as string replacement during builds so this may end up as an empty string before it even gets to user code.